Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -2044,6 +2044,8 @@ sendSmsToDriver merchantShortId opCity driverId volunteerId _req@SendSmsReq {..}
   void $ checkIfDriverSMSReceivingLimitExceeded driverId.getId transporterConfig.driverSmsReceivingLimit channel
 
   let personId = cast @Common.Driver @DP.Person driverId
+  mbVehicle <- QVehicle.findById personId
+  let mbVehicleCategory = mbVehicle >>= (.category)
```

Comment:
In most cases vehicleCategory is non in vehicle, maybe convert from vehicleVariant.